tests/browser/test/delete_blocks_test.js

```diff
@@ -135,10 +136,8 @@ suite('Delete blocks', function (done) {
   test('Delete block using backspace key', async function () {
     const before = (await getAllBlocks(this.browser)).length;
     // Get first print block, click to select it, and delete it using backspace key.
-    const block = (await getBlockElementById(this.browser, firstBlockId)).$(
-      '.blocklyPath',
-    );
-    await block.click();
+    const clickEl = await getClickableBlockElementById(this.browser, firstBlockId);
+    await clickEl.click();
```

Comment:
Instead of calling this and then calling `click` could you create a `clickBlock` helper that does this and then removes the special ID? Then it cleans up after itself, and I think the code would be easier to read. If the `clickBlock` helper takes in the same options as `click` you could also use it from the context menu code.

Reply:
Done.